src/pages/MoviesView/Movies.jsx

```diff
@@ -30,33 +32,33 @@ const MoviesView = () => {
       type: GET_MOVIES_PENDING,
       payload: { page, filter },
     });
-  }, [page, filter]);
+  }, [page, filter, dispatch]);
 
   return (
     <MoviesViewContainer>
-      {movies.length ? (
-        <>
-          <Pagination
-            page={parseInt(page, 10)}
-            totalPages={totalPages}
-            handleClick={handleChangePage}
-          />
-          <MoviesListContainer>
-            {movies.map((movie, index) => <MovieCard key={index} {...movie} />)}
-          </MoviesListContainer>
-          <Pagination
-            page={parseInt(page, 10)}
-            totalPages={totalPages}
-            handleClick={handleChangePage}
-          />
-        </>
-      )
-        : (
-          <NoResultsContainer>
-            <NoResults>No results</NoResults>
-          </NoResultsContainer>
-        )}
-
+      {isLoading ? <Spinner /> // eslint-disable-line
```

Comment:
we could split this expression to two different:
1. `{isLoading && <Spinner />}`
2. `{!!movies.length && !isLoading ? () : no data}`

what do you think am I correct?